Update EDL for Long running integration in XSOAR 8 (#29127)
demisto/content: Packs/EDL/Integrations/EDL/README.md

| List Size                          | Maximum number of items in the list.                                                                                                                                                                                                                 | True         |
| Refresh Rate                       | How often to refresh the list (e.g., less than 1 minute, 5 minutes, 12 hours, 7 days, 3 months, 1 year). For performance reasons, we do not recommend setting this value at less than 1 minute.                                                      | False        |
| Listen Port                        | Runs the service on this port from within Cortex XSOAR. Requires a unique port for each long-running integration instance. Do not use the same port for multiple instances. <br>Note: If you click the test button more than once, a failure may occur mistakenly indicating that the port is already in use.  <br> (For Cortex XSOAR 8) Only supported when used on an engine. Otherwise, this setting will be ignored.                                                            | True         |

> | Certificate (Required for HTTPS)   | For use with HTTPS - the certificate that the service should use.  <br> (For Cortex XSOAR 8) Custom certificates are not supported.                                                                                                                                                                                  | False        |
> | Private Key (Required for HTTPS)   | For use with HTTPS - the private key that the service should use.  <br> (For Cortex XSOAR 8) Configuring private API keys is supported only on engines, not on the Cortex XSOAR​​ server.                                                                                                                                                                                  | False        |
> | Username                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | False - For Cortex XSOAR 6.x <br> True - For Cortex XSOAR 8          |
> | Password                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | False - For Cortex XSOAR 6.x <br> True - For Cortex XSOAR 8          |

Author: ```suggestion
| Certificate (Required for HTTPS)   | (For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported.                                                                                                                                                                                  | False        |
| Private Key (Required for HTTPS)   | For Cortex XSOAR 6.x) For use with HTTPS - the private key that the service should use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) When using an engine, configure a private API key. Not supported on the Cortex XSOAR​​ or Cortex XSIAM server.                                                                                                                                                                                  | False        |
| Username                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | (For Cortex XSOAR 6.x) False <br> (For Cortex XSOAR 8 and Cortex XSIAM)  Optional for engines, otherwise mandatory.    |
| Password                           | Uses basic authentication for accessing the list. If empty, no authentication is enforced.                                                                                                                                                           | (For Cortex XSOAR 6.x) False <br> (For Cortex XSOAR 8 and Cortex XSIAM)  Optional for engines, otherwise mandatory.     |
```

To access the Export Indicators service by instance name, make sure ***Instance execute external*** is enabled.

1. In Cortex XSOAR, go to **Settings > About > Troubleshooting**.
2. (Cortex XSOAR 6.x only) In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
3. In a web browser, go to:

>    (For Cortex XSOAR 6.x) `https://*<demisto_address>*/instance/execute/*<instance_name>*`

Reviewer: ```suggestion
   (For Cortex XSOAR 6.x) `https://*<xsoar_address>*/instance/execute/*<instance_name>*`
```

>    (For Cortex XSOAR 8) `https://ext-*<cortex-xsiam-address>*/xsoar/instance/execute/*<instance-name>*`

Author: ```suggestion
   (For Cortex XSOAR 8) `https://ext-<XSOAR-tenant>.crtx.<region>.paloaltonetworks.com/xsoar/instance/execute/<instance-name>`
   (For Cortex XSIAM) `https://ext-<XSIAM-tenant>.crtx.<region>.paloaltonetworks.com/xsiam/instance/execute/<instance-name>`
```
